package.json

```diff
@@ -27,14 +27,17 @@
     "@sentry/browser": "^7.1.1",
     "@sentry/cli": "^2.0.2",
     "@trivago/prettier-plugin-sort-imports": "^4.1.1",
+    "@types/chai": "^4.3.12",
     "@types/dompurify": "^3.0.0",
     "@types/escape-html": "^1.0.1",
     "@types/hammerjs": "^2.0.41",
     "@types/katex": "^0.16.0",
+    "@types/mocha": "^10.0.6",
     "@types/retry": "^0.12.1",
     "@types/scroll-into-view": "^1.16.0",
     "@types/shallowequal": "^1.1.1",
     "@types/showdown": "^2.0.0",
+    "@types/sinon": "^17.0.3",
```

Comment:
One of the stated goals is to get useful IDE completion for Chai / Mocha / Sinon. Can we get those just by introducing these typing packages and adding types for the global `assert`, as a separate step from converting tests to TypeScript?

Reply:
That's actually a good idea. Maybe we can make that work.

Reply:
There https://github.com/hypothesis/client/pull/6240